src/elf/mod.rs

```diff
@@ -347,7 +324,6 @@ if_sylvan! {
                 is_64: is_64,
                 is_lib: is_lib,
                 entry: entry as u64,
-                bias: bias as u64,
```

Comment:
hmmm; I don't like to see this field go, but given now we're supporting multiple biases we have to. Or maybe I'm wrong and its useless? Still might be nice to expose the iteration macro as a function hanging off the Elf type for users to pass an unbiased value in and get biased value out?

Reply:
Since there's no single bias, this field seems wrong or at least misleading to me. Exposing the iteration might be ok, but we can wait until someone needs it. The current iteration can't be part of the Elf type because we need it during construction of the Elf type, but any exposed version would be best as part of the Elf type.